Source/C++/Core/Ap4AtomFactory.cpp

```diff
@@ -711,6 +712,13 @@ AP4_AtomFactory::CreateAtomFromStream(AP4_ByteStream& stream,
                 atom = AP4_Dec3Atom::Create(size_32, stream);
             }
             break;
+          case AP4_ATOM_TYPE_DAC3:
+            if (atom_is_large) return AP4_ERROR_INVALID_FORMAT;
+            if (GetContext() == AP4_ATOM_TYPE_AC_3) {
```

Comment:
If you want to support encrypted audio tracks, you could do so by changing this line to the following:
if (GetContext() == AP4_ATOM_TYPE_AC_3 || GetContext() == AP4_ATOM_TYPE_ENCA) {